src/newLaunch/stages/stage3.ts

```diff
@@ -28,7 +28,7 @@ export class Stage3 extends BaseStage<ILaunchConfig> {
     super(router, ethereumService, eventAggregator, tokenService);
   }
 
-  attached(): void {
+  bind(): void {
```

Comment:
Naive question: Why bind is better here?

Reply:
Reason: getter in the view triggers before `attached`, and in that getter `stage3State` is used, thus init object earlier in `bind`